### ISSUE TITLE
Unable to create Schedule

### ISSUE THREAD

**Issue by ilyapelyovin** [denouche/terraform-provider-awx#71]
Good day!

I can't create a schedule for a job template.

Code:

```
resource "awx_schedule" "run" {

  name                    = "${var.PROJECT_NAME} run"
  description             = var.FOOTPRINT
  unified_job_template_id = awx_job_template.run.id
  rrule                   = var.RUN_SCHEDULE
}
```

Plan:

```
  # awx_schedule.run will be created
  + resource "awx_schedule" "run" {
      + description             = "Terraform project: ansible/ansible-playbooks/gw-d"
      + enabled                 = true
      + id                      = (known after apply)
      + name                    = "gw-d run"
      + rrule                   = "DTSTART;TZID=Europe/Paris:20211214T120000 RRULE:INTERVAL=1;FREQ=DAILY"
      + unified_job_template_id = 74
    }
```

Result:

```
awx_schedule.run: Creating...
╷
│ Error: Unable to create Schedule
│
│   with awx_schedule.run,
│   on awx.tf line 92, in resource "awx_schedule" "run":
│   92: resource "awx_schedule" "run" {
│
│ Schedule failed to create Errors:
│ - detail: []
╵
```

AWX 23.7.0
Terraform v1.7.5
+ provider registry.terraform.io/denouche/awx v0.24.3

**Comment by lavahot** [comment]
I think you might have to include an inventory ID.

**Comment by ilyapelyovin** [comment]
Seems to be fixed in 0.27

This code works:
```
resource "awx_schedule" "run" {

  name                    = "Project name"
  description             = "Created by Terraform"
  rrule                   = "DTSTART;TZID=Europe/Paris:20211214T120000 RRULE:INTERVAL=1;FREQ=DAILY"
  unified_job_template_id = awx_job_template.run.id
}
```
